### PR TITLE
M577 Bleaching collect record form text improvements

### PR DESCRIPTION
**Pull request by mmnoo**
Requirements from [ticket](https://trello.com/c/u6dG5qte/577-suggestion-for-bleaching-su-form):
> Change the subheading Quadrat to Quadrat Collection
Change the help text for the Number field to Number is automatically generated by MERMAID as an identifier for a bleaching sample unit, and is not editable.

I put some other text I bumped into into the language file while I was at it.

### CODE REVIEW

**Comment by mmnoo** [comment]
[Suggestion for Bleaching SU Form](https://trello.com/c/u6dG5qte/577-suggestion-for-bleaching-su-form)

**Review by alanjleonard — APPROVED** [review]

File: src/language.js [hunk 1]
```diff
@@ -284,6 +284,12 @@ const pages = {
     totalAbundanceLabel: 'Total Abundance',
     totalBiomassLabel: 'Total Biomass (kg/ha)',
     newBenthicAttributeLink: 'Propose New Benthic Attribute...',
+    formSectionTitle: {
+      sampleEvent: 'Sample Event',
+      quadrat: 'Quadrat Collection',
```
Comment (saanobhaai) [inline src/language.js:289]: Can we make the key `quadratcollection`? A small thing, but the distinction is what led to the current user confusion in the first place, so best to be consistent.

**Review by saanobhaai — CHANGES_REQUESTED** [review]
One small change request

File: src/language.js (hunk 1 above)
Comment (mmnoo) [reply src/language.js]: changed.

**Pull request merged**